gems/aws-sdk-s3/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Unreleased Changes
 ------------------
 
+1.119.3 (2023-03-30)
```

Comment:
Can remove this line - the `* issue - ` line should just be under "Unreleased Changes".  The release system will take all unreleased features/issues and create a release + make the appropriate updates to Changelog/Version